config/production.json

```diff
@@ -14,5 +14,8 @@
 			"idp_url": "https://auth.idp.hashicorp.com"
 		},
 		"product_slugs_with_integrations": ["vault", "waypoint", "nomad"]
+	},
+	"flags": {
+		"enable_unified_search": false
```

Comment:
This is redundant with the same setting in `base.json`, but felt worth adding as a safeguard to ensure half-finished unified search work never appears in production.